res/controllers/Intech TEK2.scripts.js

```diff
@@ -0,0 +1,32 @@
+const TEK2 = {};
+TEK2.jogScratchSensitivity = 1;
+TEK2.jogPitchSensitivity = 3;
+
+TEK2.init = function() {};
+
+TEK2.shutdown = function() {};
+
+// The button that enables/disables scratching
+TEK2.wheelTouch = function(channel, control, value, status, group) {
+    const deckNumber = script.deckFromGroup(group);
+    if (value === 0x7f) {
+    // If button downs
+        const alpha = 1.0 / 8;
+        const beta = alpha / 32;
+        engine.scratchEnable(deckNumber, 128, 33 + 1 / 3, alpha, beta);
```

Comment:
128 seems quite low-res... Are you sure thats how many ticks there are in a full rotation?

Reply:
I adjusted the Endless Potmeter in the Grid Editor, and it's worked nicely with this value.

Reply:
So if with the current value, doing one full rotation of physical encoder matches one full rotation of the vinyl spinny widget?

Reply:
Good point. I made some adjustments, and now it matches both Scratch and Simple Turn.